client/client.go

```diff
@@ -136,6 +149,48 @@ func loadTemplate(name string) *expander.Template {
 	return template
 }
 
+func buildTemplateFromType(name string, typeName string) *expander.Template {
+	// TODO: Fill this in with an actual URL fetched from github.
```

Comment:
For this one I was thinking of having a type represented as:
Type {
  name string
  version string
}

just in case there are other things that we need to bake in later. So, perhaps use that here. I can obviously change this (or construct the name that I pass in here) once I get my stuff in. Just an FYI.